src/sentry/search/utils.py

```diff
@@ -673,10 +711,10 @@ def parse_query(projects, query, user, environments):
 
     results["query"] = " ".join(results["query"])
 
-    return results
+    return results  # type: ignore[return-value]
```

Comment:
You added a type for the return value, and then you are ignoring it here.

Reply:
Yeah, I get this `Incompatible return value type (got "Dict[str, Any]", expected "ParsedQueryValues")` which makes sense since `Dict[str, Any]` is more ambiguous than `ParsedQueryValues(TypedDict)`

Would casting the generic dict to `ParsedQueryValues` make sense here?